src/ExternalSearch.Providers.CompanyHouse/Constants.cs

```diff
@@ -96,13 +96,13 @@ public static class Constants
 
         public struct KeyName
         {
-            public const string ApiKey = nameof(ApiKey);
-            public const string AcceptedEntityType = nameof(AcceptedEntityType);
-            public const string CompanyHouseNumberKey = nameof(CompanyHouseNumberKey);
-            public const string CountryKey = nameof(CountryKey);
-            public const string OrgNameKey = nameof(OrgNameKey);
-            public const string SkipCompanyHouseNumberEntityCodeCreation = nameof(SkipCompanyHouseNumberEntityCodeCreation);
-            public const string SkipCompanyHouseNameEntityCodeCreation = nameof(SkipCompanyHouseNameEntityCodeCreation);
+            public const string ApiKey = "apiKey";
```

Comment:
the old nameof(...) would have returned the value AcceptedEntityType not the camel case that this new version returns. Have you tested if this will cause an issue for a customer upgrading?

Reply:
tested, it's case insensitive